network-api/networkapi/wagtailpages/pagemodels/libraries/rcc/detail_page.py

```diff
@@ -17,10 +16,9 @@
 from networkapi.wagtailpages.pagemodels.customblocks.base_rich_text_options import (
     base_rich_text_options,
 )
-
-# from networkapi.wagtailpages.pagemodels.libraries.rcc import authors_index
-# from networkapi.wagtailpages.pagemodels.profiles import Profile
-# from networkapi.wagtailpages.utils import localize_queryset
+from networkapi.wagtailpages.pagemodels.libraries.rcc import authors_index
+from networkapi.wagtailpages.pagemodels.profiles import Profile
+from networkapi.wagtailpages.utils import localize_queryset
```

Comment:
The imports here seem to diverge from the module level style imports that are used above. Can you add some context why, if this was intentional?

Reply:
Honestly, this was just copied over. I'll adjust that